packages/preview-middleware-client/src/adp/quick-actions/table-quick-action-base.ts

```diff
@@ -103,8 +112,11 @@ export abstract class TableQuickActionDefinitionBase {
             if (section) {
                 this.collectChildrenInSection(section, table);
             } else if (this.iconTabBar && tabKey) {
+                let tableRows = this.getInternalTableRows(table);
                 this.children.push({
                     label: `'${iconTabBarfilterMap[tabKey]}' table`,
+                    tooltip: (tableRows.length < 1 && this.tooltip) ? this.context.resourceBundle.getText(this.tooltip) : undefined,
+                    enabled: !(tableRows.length < 1 && this.tooltip),
```

Comment:
Why table rows count affects `enabled` property in the base class? It is action specific condition. Why tooltip is considered here?